pkg/azurefile/azurefile.go

```diff
@@ -197,7 +195,8 @@ var (
 
 	retriableErrors = []string{accountNotProvisioned, tooManyRequests, shareBeingDeleted, clientThrottled}
 
-	defaultAzcopyCopyOptions = []string{"--recursive", "--check-length=false"}
+	defaultAzcopyCopyOptions  = []string{"--recursive", "--check-length=false"}
```

Comment:
this is volume cloning options? pls add comments why `check-length=false` in cloning why it's true in snapshot options

Reply:
done